cql3/statements/alter_keyspace_statement.cc

```diff
@@ -144,15 +144,14 @@ void add_prefixed_key(const sstring& prefix, const std::map<sstring, sstring>& i
 };
 
 std::map<sstring, sstring> get_current_options_flattened(const shared_ptr<cql3::statements::ks_prop_defs>& ks,
-                                                         bool include_tablet_options,
                                                          const gms::feature_service& feat) {
     std::map<sstring, sstring> all_options;
 
     add_prefixed_key(ks->KW_REPLICATION, ks->get_replication_options(), all_options);
     add_prefixed_key(ks->KW_STORAGE, ks->get_storage_options().to_map(), all_options);
     // if no tablet options are specified in ATLER KS statement,
     // we want to preserve the old ones and hence cannot overwrite them with defaults
-    if (include_tablet_options) {
+    if (ks->get_map(ks->KW_TABLETS).has_value()) {
```

Comment:
nit: get_map is too expensive just for checking if the property exists.
you should be able to use `has_property(ks->KW_TABLETS)` instead which is much cheaper.

Reply:
Definitely. Fixed.